### PR TITLE
Add Yabai extension

### PR DESCRIPTION
**Pull request by krzysztoff1**
## Description

Hi, I'm sharing the extension I've been using daily for a few months now. It adds basic integration of Yabai window manager commands. I'm aware of previous pr adding this extension, hope this one resolves issues mentioned in it.

## Screencast

https://github.com/raycast/extensions/assets/78202674/ceba9e28-8c51-4ccf-9969-8bf0b615be79

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
There is also a [newer extension](https://github.com/raycast/extensions/pull/9965) in the queue, could we agree as a base and move the features from one of them to the other one to get only one Yabai extension?

**Comment by krzysztoff1** [comment]
Hi @rosenpin! Yes migrating to my pr would be great. Feel free to open a pr on my fork. Thanks!

**Comment by rosenpin** [comment]
Unfortunately I won't have much time to do it in the coming weeks. You are more than welcome to use the code from my repo and add it to yours if you can.  
Otherwise, hopefully I'll be able to get around to it in a few weeks or so

File: extensions/yabai/package.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    {
+      "name": "yabaiExecutablePath",
+      "description": "Path to the yabai executable.",
+      "type": "textfield",
+      "required": false,
+      "default": "./opt/homebrew/bin/yabai",
+      "title": "Path to the yabai executable",
+      "placeholder": "./opt/homebrew/bin/yabai",
+      "label": "By default, the extension will use the yabai executable from the homebrew installation. If you have yabai installed in a different location, you can specify it here."
+    }
```
Comment (erics118) [inline extensions/yabai/package.json:98]: Should the `.` be there? Also, this doesn't work on Intel macs. I suggest doing something similar to:

https://github.com/raycast/extensions/blob/main/extensions/brew-services/package.json#L26
https://github.com/raycast/extensions/blob/main/extensions/brew-services/src/services.tsx#L28-L33

Comment (krzysztoff1) [reply extensions/yabai/package.json:95]: Thanks! You are right, should be better now

**Comment by krzysztoff1** [comment]
@pernielsentikaer The pr is ready for review. I added the commands from @rosenpin.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**